<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreDialog</class>
 <widget class="QDialog" name="PreDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1084</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Start">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Continue">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>370</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>-10</y>
     <width>401</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;Magneto&quot;;</string>
   </property>
   <property name="text">
    <string>Character Configure</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cha_soldier">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>90</y>
     <width>190</width>
     <height>190</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/map/character0_fullsize.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="cha_magician">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>90</y>
     <width>190</width>
     <height>190</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/map/character1_fullsize.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="save_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>当前未载入存档，选择角色并点击Start开始</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>300</y>
     <width>241</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QLabel" name="c1_s">
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="c1_atk_r">
      <property name="text">
       <string>360</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="c1_atk_f">
      <property name="text">
       <string>1000</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="c1_d">
      <property name="text">
       <string>20</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>速度：</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="c1_atk_s">
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>射程：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>生命上限：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>攻击力：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="c1_h">
      <property name="text">
       <string>350</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>攻击频率：</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>子弹速度：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="c1_h_enhance">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="c1_d_enhance">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="c1_s_enhance">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="c1_atk_f_enhance">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="c1_atk_r_enhance">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="c1_atk_s_enhance">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>300</y>
     <width>231</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="5" column="1">
     <widget class="QLabel" name="c2_atk_s">
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>生命上限：</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>子弹速度：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>速度：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="c2_atk_r">
      <property name="text">
       <string>400</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="c2_s">
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>射程：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="c2_d">
      <property name="text">
       <string>30</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>攻击力：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="c2_atk_f">
      <property name="text">
       <string>800</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>攻击频率：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="c2_h">
      <property name="text">
       <string>280</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="c2_h_enhance">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="c2_d_enhance">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="c2_s_enhance">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="c2_atk_f_enhance">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="c2_atk_r_enhance">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="c2_atk_s_enhance">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="money">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>持有金币：0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>640</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>金币系数</string>
   </property>
  </widget>
  <widget class="QPushButton" name="money_k_enhance">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>640</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLabel" name="money_k">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>640</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>200</string>
   </property>
  </widget>
  <widget class="QLabel" name="lives">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>640</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lives_enhance">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>640</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>640</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>复活次数</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
